x-pack/plugins/apm/e2e/ci/kibana.e2e.yml

```diff
@@ -11,7 +11,7 @@ elasticsearch.hosts: http://localhost:9201
 elasticsearch.username: 'kibana_system_user'
 elasticsearch.password: 'changeme'
 
-# APM index pattern
+# APM data view
 apm_oss.indexPattern: apm-*
```

Comment:
@sqren Should I also change the config key to dataView?

Reply:
It's been deprecated so you can just leave that: https://github.com/elastic/kibana/pull/111133

Reply:
Btw. weren't you going to remove the entire `e2e` folder?

Reply:
I will, but in another PR.